scrooge-generator/src/main/resources/nodegen/service.mustache

```diff
@@ -90,17 +89,16 @@ export class Client {
 }
 
 export class Processor {
-    private _handler: any
+    private _handler
```

Comment:
This change was a consequence of reverting a commit that broke the generator. I agree we should add types wherever applicable though.